src/components/ChangePlan/Checkout/PaymentMethod/PaymentMethod.js

```diff
@@ -4,150 +4,188 @@ import { useIntl } from 'react-intl';
 import { Form, Formik } from 'formik';
 import { Loading } from '../../../Loading/Loading';
 import Cards from 'react-credit-cards';
-import { fakePaymentMethodInformation } from '../../../../services/doppler-billing-user-api-client.double';
-import { fakeAccountPlanDiscounts } from '../../../../services/doppler-account-plans-api-client.double';
-import useTimeout from '../../../../hooks/useTimeout';
+import queryString from 'query-string';
+import { useLocation } from 'react-router';
+import { getFormInitialValues, extractParameter } from '../../../../utils';
 import { FieldGroup, FieldItem } from '../../../form-helpers/form-helpers';
 import { Discounts } from '../Discounts/Discounts';
-import { getFormInitialValues } from '../../../../utils';
 
 const fieldNames = {
   paymentMethodName: 'paymentMethodName',
 };
 
+const paymentMethods = [
+  {
+    value: 'CC',
+    description: 'checkoutProcessForm.payment_method_credit_card_method',
+  },
+  {
+    value: 'TRANSF',
+    description: 'checkoutProcessForm.payment_method_transfer_method',
+  },
+  {
+    value: 'MP',
+    description: 'checkoutProcessForm.payment_method_mercado_pago_method',
+  },
+];
+
 const paymentType = {
-  MercadoPago: 'MP',
+  creditCard: 'CC',
+  mercadoPago: 'MP',
 };
 
-export const PaymentMethod = InjectAppServices(({ dependencies: { appSessionRef }, showTitle }) => {
-  const intl = useIntl();
-  const [state, setState] = useState({ loading: true });
-  const createTimeout = useTimeout();
+const creditCardType = {
+  mastercard: 'Mastercard',
+  visa: 'Visa',
+  amex: 'American Express',
+};
 
-  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+const amexDescription = 'amex';
 
-  useEffect(() => {
-    const fetchData = async () => {
-      const sessionPlan = appSessionRef.current.userData.user;
-      const discounts = fakeAccountPlanDiscounts;
+const getCreditCardBrand = (ccType) => {
+  // check for American Express
+  if (ccType === creditCardType.amex) {
+    return amexDescription;
```

Comment:
Why are you doing this? It's a bit weird...

Reply:
I think it's because of react-credit-card, I'm not sure. 
![image](https://user-images.githubusercontent.com/84402180/131372632-c468b860-d35d-4358-9662-3fef303aa14d.png)

Reply:
this function maybe not be required. I think it would be enough just to pass `ccType`, can it be @jhoffman-ms  ?

Reply:
Yes, exactly. The reason of this is the API returns "American Express" and the credit-card component supports 'amex'

Reply:
> this function maybe not be required. I think it would be enough just to pass `ccType`, can it be @jhoffman-ms ?

The function is neccesary because the API returns "American Express" and the component needs 'amex'

Reply:
Thanks for the clarification!.